### PR TITLE
Clarify documentation and argument names

### PR DESCRIPTION
**Pull request by 1lann**
Correct grammatical errors with the documentation, and change parts where the documentation and argument names will state "message" when it actually means "event".

### CODE REVIEW

File: adapter.go [hunk 1]
```diff
@@ -1,16 +1,16 @@
 package socketio
 
-// BroadcastAdaptor is the adaptor to handle broadcast.
+// BroadcastAdaptor is the adaptor to handle broadcasts.
 type BroadcastAdaptor interface {
 
-	// Join lets socket join the t room.
+	// Join lets the socket join a room.
```
Comment (googollee) [inline adapter.go]: `join the room` means the room which presented by input parameter `room`.

File: handler.go [hunk 2]
```diff
@@ -19,13 +19,13 @@ func newBaseHandler(name string, broadcast BroadcastAdaptor) *baseHandler {
 	}
 }
 
-// On registers the function f to handle message.
-func (h *baseHandler) On(message string, f interface{}) error {
+// On registers the function f to handle an event.
+func (h *baseHandler) On(event string, f interface{}) error {
```
Comment (googollee) [inline handler.go]: `message` is the word used by JS implementation.

Comment (1lann) [inline handler.go]: Looking at the examples here: http://socket.io/docs/, they call the first argument an event, but the entire package (event and payload) a message.

Comment (mcastilho) [inline handler.go]: I agree that socket.io call it "event".

**Comment by googollee** [comment]
OK, agree with you all.

**Pull request merged**